src/pages/RoomMembersPage.tsx

```diff
@@ -78,6 +78,13 @@ function RoomMembersPage({report, session, policies}: RoomMembersPageProps) {
         [],
     );
 
+    useEffect(() => {
+        if (isFocusedScreen) {
+            return;
+        }
+        setSelectedMembers([]);
+    }, [isFocusedScreen]);
```

Comment:
Clearing the selection is an unwanted behaviour here. As for the bug this is fixing, we should have just updated the list. Coming from https://github.com/Expensify/App/issues/59426